assets/src/blocks/Counter/CounterBlock.js

```diff
@@ -81,11 +78,16 @@ export class CounterBlock {
           name: 'arc',
           label: __( 'Progress Dial', 'planet4-blocks' )
         },
-        {
-          name: 'en-forms-bar',
-          label: __( 'Progress Bar inside EN Form', 'planet4-blocks' )
-        }
+
       ]
-    );
+
+    if (!window.p4ge_vars.planet4_options.feature_engaging_networks) {
```

Comment:
This condition need to reverse. when EN feature is enabled then only EN form counter block style is available.